forge.tf

```diff
@@ -0,0 +1,21 @@
+resource "azurerm_container_registry" "forge" {
+  name                = replace("${var.resource_group}forge", "/[^a-zA-Z0-9]/", "")
```

Comment:
Not 100% sure exposing forge is the best over something generic, but there's an argument no customer will ever see it, so I'm not sure I care much either way.

Reply:
I can switch it to some UUID or random characters. The name must be globally unique which is just annoying.

Reply:
switched to domino for now